specification/ai/DocumentIntelligence/client.tsp

```diff
@@ -53,6 +53,38 @@ interface DocumentIntelligenceAdministrationClient {
   deleteClassifier is DocumentClassifiers.deleteClassifier;
 }
 
+@@encodedName(AnalyzeDocumentRequest, "csharp", "AnalyzeDocumentContent");
```

Comment:
encodedName is for serialization purpose. Use clientName to change model name.

Reply:
Thank you. This fixed the first problem but I got back to the issue I was having initially.
> /mnt/vss/_work/1/s/specification/ai/DocumentIntelligence/client.tsp:62:34 - error invalid-ref: Union doesn't have member EAN13
 62 | @@clientName(DocumentBarcodeKind.EAN13, "csharp", "Ean13");

This seems to happen because some model properties are declared as strings:
```
union DocumentBarcodeKind {
  string,

  @doc("QR code, as defined in ISO/IEC 18004:2015.")
  "QRCode",

  @doc("PDF417, as defined in ISO 15438.")
  "PDF417",

  @doc("GS1 12-digit Universal Product Code.")
  "UPCA",

  @doc("GS1 6-digit Universal Product Code.")
  "UPCE",

  @doc("Code 39 barcode, as defined in ISO/IEC 16388:2007.")
  "Code39",

  @doc("Code 128 barcode, as defined in ISO/IEC 15417:2007.")
  "Code128",

  @doc("GS1 8-digit International Article Number (European Article Number).")
  "EAN8",

  @doc("GS1 13-digit International Article Number (European Article Number).")
  "EAN13",

  @doc("GS1 DataBar barcode.")
  "DataBar",

  @doc("Code 93 barcode, as defined in ANSI/AIM BC5-1995.")
  "Code93",

  @doc("Codabar barcode, as defined in ANSI/AIM BC3-1995.")
  "Codabar",

  @doc("GS1 DataBar Expanded barcode.")
  "DataBarExpanded",

  @doc("Interleaved 2 of 5 barcode, as defined in ANSI/AIM BC2-1995.")
  "ITF",

  @doc("Micro QR code, as defined in ISO/IEC 23941:2022.")
  "MicroQRCode",

  @doc("Aztec code, as defined in ISO/IEC 24778:2008.")
  "Aztec",

  @doc("Data matrix code, as defined in ISO/IEC 16022:2006.")
  "DataMatrix",

  @doc("MaxiCode, as defined in ISO/IEC 16023:2000.")
  "MaxiCode",
}
```

Do you know what's the right way of renaming those?

Reply:
What you defined for the value is anonymous, which means it doesn't have a name so that you cannot access it by name. An alternative is to define your union like:
```
union DocumentBarcodeKind {
  string,
  EAN13: "EAN13",
}
```
and change it by
```
@@clientName(DocumentBarcodeKind.EAN13, "Ean13");
```